crates/neon/src/lib.rs

```diff
@@ -98,3 +98,6 @@ pub use neon_macros::*;
 
 #[cfg(feature = "napi-6")]
 mod lifecycle;
+
+#[cfg(feature = "napi-8")]
+static MODULE_TAG: crate::sys::TypeTag = include!(concat!(env!("OUT_DIR"), "/type_tag.rs"));
```

Comment:
We only need a single tag because Neon only stores a single type `Box<dyn Any>`. However, as a future optimization, without any breaking changes, we could use [`TypeId`](https://doc.rust-lang.org/stable/std/any/struct.TypeId.html) as part of the definition and avoid an extra layer of indirection--implementing our *own* `Any`. However this would rely on the internal representation of `TypeId` which is not part of Rust's stable API.